{
	"id": "oval:com.redhat.cve:def:202438875",
	"version": "636",
	"class": "vulnerability",
	"metadata": {
		"title": "python-django: Potential denial-of-service in django.utils.html.urlize() (moderate)",
		"reference": [
			{
				"ref_id": "CVE-2024-38875",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2024-38875",
				"source": "CVE"
			}
		],
		"description": "DOCUMENTATION: A vulnerability was found in the Python-Django in django.utils.html.urlize() function. The issue triggers when it is parsed with certain inputs with a very large number of brackets, leading to a potential denial of service attack.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2024-07-26"
			},
			"cve": [
				{
					"text": "CVE-2024-38875",
					"cvss3": "5.3/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
					"cwe": "CWE-1287",
					"href": "https://access.redhat.com/security/cve/CVE-2024-38875",
					"impact": "moderate",
					"public": "20240726:0626"
				}
			],
			"affected": {
				"resolution": [
					{
						"state": "Affected",
						"component": [
							"python-django",
							"python2-django"
						]
					}
				]
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:satellite:6.10",
					"cpe:/a:redhat:satellite:6.10::el7",
					"cpe:/a:redhat:satellite_capsule:6.10",
					"cpe:/a:redhat:satellite_capsule:6.10::el7"
				]
			},
			"issued": {}
		},
		"affected": {}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844003",
										"comment": "python2-django is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844004",
										"comment": "python2-django is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844001",
										"comment": "python-django is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844002",
										"comment": "python-django is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.cve:tst:20134120009",
						"comment": "Red Hat Enterprise Linux 7 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.cve:tst:20134120010",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
